vfs/src/lib.rs

```diff
@@ -48,6 +48,9 @@ impl ReadOnlyVfs {
     pub fn len(&self) -> usize {
         self.data.len()
     }
+    pub fn is_empty(&self) -> bool {
+        self.data.is_empty()
+    }
```

Comment:
What prompted this, I'm curious.

Reply:
due to the clippy lint https://rust-lang.github.io/rust-clippy/master/index.html#len_without_is_empty